migrations/migrations/20191120170031-create-user-albums.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) =>
```

Comment:
user_id and album_id must be primary keys

Reply:
Changed in latest commit!